cognite/client/_api/data_modeling/instances.py

```diff
@@ -623,6 +623,24 @@ def apply(
                 ...     auto_create_end_nodes=True
                 ... )
 
+            Make node apply using helper functions:
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes.data_modeling import NodeApply
+                >>> from cognite.client.utils import datetime_to_ms_iso_timestamp
+                >>> from datetime import datetime
+                >>> my_date = datetime(2020, 3, 14, 15, 9, 26, 535000)
+                >>> client = CogniteClient()
+                >>> node = NodeApply(
+                ...     space="actors",
+                ...     external_id="arnold_schwarzenegger",
+                ...     sources=[
+                ...         NodeOrEdgeData(
+                ...             ViewId("mySpace", "PersonView", "v1"),
+                ...             {"name": "Piday", "date": datetime_to_ms_iso_timestamp(my_date)}
```

Comment:
I just saw that the number of examples here is quite lengthy already, should we do something like just showcase the `datetime_to_ms_iso_timestamp` function?

Reply:
Something like this?